### PR TITLE
Add test for Series.shape

### PR DESCRIPTION
**Pull request by beobest2**
Adding missing test for Series.shape

스크린샷 2020-05-12 오전 9 47 50

### CODE REVIEW

File: databricks/koalas/tests/test_series.py [hunk 1]
```diff
@@ -1489,3 +1489,16 @@ def test_floordiv_nan(self):
         kser = ks.from_pandas(pser)
 
         self.assert_eq(repr(pser.floordiv(np.nan)), repr(kser.floordiv(np.nan)))
+
+    def test_shape(self):
+        kser = ks.Series(["a", "b", "c"])
+        pser = kser.to_pandas()
+
+        self.assert_eq(repr(pser.shape), repr(kser.shape))
```
Comment (ueshin) [inline databricks/koalas/tests/test_series.py:1497]: Do we need to use `repr`?

**Comment by HyukjinKwon** [comment]
@beobest2 can you rebase this to resolve the conflicts here? Looks good otherwise.

File: databricks/koalas/tests/test_series.py (hunk 1 above)
Comment (HyukjinKwon) [reply databricks/koalas/tests/test_series.py:1497]: Yeah, everything here seems a string. So looks we won't need. Can you double check @beobest2?  Let's fix the one in https://github.com/databricks/koalas/commit/2c615ad076a262b4a3efaddca4eaa62415d6edb2 too if this is to fix.

**Comment by beobest2** [comment]
> @beobest2 can you rebase this to resolve the conflicts here? Looks good otherwise.

I have fixed the confilcts and remove `repr` on both `test_shape` and `test_to_frame` on test_series.py.
Please check

**Comment by HyukjinKwon** [comment]
@beobest2, seems the conflict was tried to resolve by manually picking the commits. It should either be `git merge` or `git rebase` so the PR doesn't include duplicated changes picked from the commits.

**Comment by beobest2** [comment]
@HyukjinKwon 
PR has been modified to not include duplicate changes selected in the commit. I will be careful next time. Thank you

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**